modelforge/dataset/dataset.py

```diff
@@ -189,12 +202,12 @@ def _from_hdf5(self) -> None:
                                     not_nan = False
                                     break
                             else:
-                                #I don't like this, it is not general, but necessary at this point to
+                                # I don't like this, it is not general, but necessary at this point to
```

Comment:
You could probably remove my comment :-)

Reply:
:-) removed